FWCore/Integration/test/WhatsItESProducer.cc

```diff
@@ -66,27 +68,25 @@ class WhatsItESProducer : public edm::ESProducer {
 // constructors and destructor
 //
 WhatsItESProducer::WhatsItESProducer(edm::ParameterSet const& pset)
-: dataLabel_(pset.exists("doodadLabel")? pset.getParameter<std::string>("doodadLabel"):std::string(""))
+  : dataLabel_{pset.exists("doodadLabel") ? pset.getParameter<std::string>("doodadLabel"): std::string{}}
```

Comment:
I do not think there is a need for `dataLabel_` since `token_` is doing the job now.

Reply:
Agreed.